_data/categories.json

```diff
@@ -24,6 +24,11 @@
     "title": "Communication",
     "icon": "fas fa-comments"
   },
+  {
+    "name": "crowdfunding",
+    "title": "Crowdfunding",
+    "icon": "fas fa-handshake"
```

Comment:
```suggestion
    "icon": "fas fa-hand-holding-usd"
```
The `hand-holding-usd` icon may work slightly better?
![image](https://user-images.githubusercontent.com/20560218/123517865-4886c980-d69b-11eb-8402-940286ea0e79.png)